tests/integration/test_store.py

```diff
@@ -2081,6 +2106,30 @@ def test_span_ingestion_with_performance_scores(
             ),
         )
     )
+    envelope.add_item(
+        Item(
+            type="span",
+            payload=PayloadRef(
+                bytes=json.dumps(
+                    {
+                        "data": {
+                            "transaction": "/page/with/click/interaction/jane/123"
```

Comment:
`"123"` will be scrubbed by the generic scrubber, `"jane"` will be scrubbed by the project config rule defined in this test.